mesa/experimental/cell_space/cell.py

```diff
@@ -131,10 +131,34 @@ def is_full(self) -> bool:
     def __repr__(self):  # noqa
         return f"Cell({self.coordinate}, {self.agents})"
 
+    @property
+    @cache
+    def neighborhood(self) -> CellCollection:
+        """Returns the direct neigborhood of the cell
+
+        This is equivalent to cell.get_neighborhood(radius=1)
+
+        """
+        return self.get_neighborhood(radius=1)
```

Comment:
The `radius=1` is redundant right? I think it's more powerful if it uses the fully default get_neighborhood() method and is thus equivalent at calling that.
```suggestion
        return self.get_neighborhood()
```